tfx/orchestration/pipeline.py

```diff
@@ -376,7 +376,7 @@ def _set_components(self, components: List[base_node.BaseNode]) -> None:
       if component.id in node_by_id:
         raise RuntimeError(
             f'Duplicated node_id {component.id} for component type'
-            f'{component.type}.')
+            f'{component.type}. Try setting a different node_id using `.with_id()`.')
```

Comment:
Could you break this line to make it fit in 80 column?

We need to follow the [style guide](https://google.github.io/styleguide/pyguide.html#32-line-length).

Reply:
Thanks, updated